package.json

```diff
@@ -109,6 +110,7 @@
     "node": "10.* || >= 12"
   },
   "dependencies": {
+    "ember-in-viewport": "^3.7.5",
```

Comment:
@brechtPhilips @ValenberghsSven De gewoonte is om addons steeds in `devDependencies` onder te brengen. Zie [hier](https://stackoverflow.com/questions/51529335/ember-package-json-dependencies-vs-devdependencies).